naucse/utils/models.py

```diff
@@ -1,8 +1,36 @@
 from collections import OrderedDict
 from pathlib import Path
-import sys
 
 import yaml
+from arca import Task, Arca
+from werkzeug.local import LocalProxy
+
+import naucse.utils.routes
+
+_arca = None
+
+
+@LocalProxy
+def arca():
+    global _arca
+    if _arca is not None:
+        return _arca
+
+    _arca = Arca(settings={"ARCA_BACKEND": "arca.backend.CurrentEnvironmentBackend",
+                           "ARCA_BACKEND_CURRENT_ENVIRONMENT_REQUIREMENTS": "requirements.txt",
+                           "ARCA_BACKEND_VERBOSITY": 2,
+                           "ARCA_BACKEND_APK_DEPENDENCIES": ["libffi-dev"],
```

Comment:
Are we still using apk? I am a bit behind here, so bare with me.

Also a comment about why libffi-dev is needed would be nice.